cartridge/confapplier.lua

```diff
@@ -487,6 +488,10 @@ local function init(opts)
         log.info('Remote control bound to 0.0.0.0:%d', vars.binary_port)
     end
 
+    if vars.box_opts.custom_proc_title ~= nil then
```

Comment:
move it to `cartridge.lua`, right after the `box_opts.custom_proc_title` assignment